docs/spec_draft.md

```diff
@@ -339,6 +355,9 @@ syntax.
   floating-point constants of `f32` or `f64` types, e.g. `(12.34, 56.78)`,
   where the first constant is the real part, and the second constant is the
   imaginary part.
+  * **String constants** String constants are represented as a sequence of
+  bytes enclosed in double quotation mark symbols, e.g. "foo123?" (in ASCII
+  encoding) or "\18\A3" (in hex encoding).
```

Comment:
Let's move string constants to the bottom of the list to reflect the proposed order in the Types section.